db/system_keyspace.hh

```diff
@@ -182,6 +182,7 @@ public:
     static constexpr auto CDC_GENERATIONS_V3 = "cdc_generations_v3";
     static constexpr auto TABLETS = "tablets";
     static constexpr auto SERVICE_LEVELS_V2 = "service_levels_v2";
+    static constexpr auto VIEW_BUILD_STATUS_V2 = "view_build_status_v2";
```

Comment:
I donno about this - sounds like a potential source of confusion to me. No one will remember which is the new and which is the old one.

Reply:
I'm ok with the `v2` but I don't know, I don't have a strong opinion about this

Reply:
In case of the auth tables I think there was an idea to deprecate the old ones and skip creating them in new clusters, maybe it would even be possible to delete old ones in upgraded clusters at some point.

However, because we keep the old view_build_status for compatibility, this argument doesn't have as much power so maybe having the v2 suffix to differentiate two tables more easily makes more sense.